octobot_tentacles_manager/cli.py

```diff
@@ -218,6 +223,8 @@ def register_tentacles_manager_arguments(tentacles_parser) -> None:
                                                          "package): -e myTentaclesFolder OctoBot-Default-Tentacles "
                                                          "-d ./tentacles", nargs="+")
     tentacles_parser.add_argument("-a", "--all", help="Apply command to all available Tentacles.", action='store_true')
+    tentacles_parser.add_argument("--export-with-package-name", help="Export package with its artifact name",
```

Comment:
I think this would be clearer `Export package using its artifact's name as package name`